src/battle/overlay_12_0224E4FC.c

```diff
@@ -2825,16 +2959,16 @@ int CheckAbilityActive(BattleSystem *bsys, BATTLECONTEXT *ctx, int flag, int bat
 //FIXME: Function name is wrong
 BOOL BattleCtx_IsIdenticalToCurrentMove(BATTLECONTEXT *ctx, int moveNo) {
```

Comment:
enough context to change this name? or not yet?

Reply:
Uhhh I'll figure something out